rhel8/profiles/stig.profile

```diff
@@ -0,0 +1,36 @@
+documentation_complete: true
+
+title: '[DRAFT] DISA STIG for Red Hat Enterprise Linux 8'
+
+description: |-
+    This profile contains configuration checks that align to the
+    [DRAFT] DISA STIG for Red Hat Enterprise Linux 8.
+
+    In addition to being applicable to Red Hat Enterprise Linux 8, DISA recognizes this
+    configuration baseline as applicable to the operating system tier of
+    Red Hat technologies that are based on Red Hat Enterprise Linux 8, such as:
+
+    - Red Hat Enterprise Linux Server
+    - Red Hat Enterprise Linux Workstation and Desktop
+    - Red Hat Enterprise Linux for HPC
+    - Red Hat Storage
+    - Red Hat Containers with a Red Hat Enterprise Linux 8 image
+
+extends: ospp
+
+selections:
+    - login_banner_text=dod_banners
+    - dconf_gnome_banner_enabled
+    - banner_etc_issue
+    - accounts_password_set_min_life_existing
+    - accounts_password_set_max_life_existing
+    - account_disable_post_pw_expiration
+    - account_temp_expire_date
+    - audit_rules_usergroup_modification_passwd
+    - sssd_enable_smartcards
+    - sssd_offline_cred_expiration
+    - smartcard_configure_cert_checking
+    - encrypt_partitions
+    - sysctl_net_ipv4_tcp_syncookies
+    - package_openssh-server_installed
```

Comment:
NACK. Baselines should not mandate the installation of SSH.

Reply:
A lot of our checks and remediations check if openssh is installed and install if it isn't. I understand your comment about not "requiring" ssh, but why not add the rule and have it tailored out for those that don't use ssh?

Reply:
Talked with @shawndwells offline and removed the rule.